SoftLayer/managers/network.py

```diff
@@ -779,16 +779,16 @@ def cancel_item(self, identifier, cancel_immediately,
                                 customer_note,
                                 id=identifier)
 
-    def get_pods(self, datacenter=None):
+    def get_pods(self, mask=None, filter=None, datacenter=None):
```

Comment:
`filter` is actually a python3 reserved word ( a built-in function in this case https://docs.python.org/3/library/functions.html#filter ) which is why a lot of the times you see `_filter`. I dont mind changing `_filter` to something else, it just can't be `filter`